src/v2/openapi.yaml

```diff
@@ -0,0 +1,740 @@
+              type: array
+              description: Private vulnerability details with limited info
+              items:
+                $ref: '#/components/schemas/BasicVulnerabilityFields'
+          description: Details of the package for free tier users
+
+    RecommendedPackageData:
+      title: RecommendedPackageData
+      description: Recommended packages data
+      allOf:
+      - $ref: '#/components/schemas/PackageDetails'
+      - type: object
+        properties:
+          confidence_reason:
+            type: number
+            format: float
+            example: 83.16431
+          reason:
+            type: string
+          topic_list:
+            type: array
+            items:
+              type: string
+              
+    RegistrationStatus:
+      title: RegistrationStatus
+      type: string
+      enum:
+        - registered
+        - freetier
+      default: freetier
+
+    StackRecommendationResult:
+      title: StackRecommendation
+      type: object
+      required:
+        - _audit
+        - uuid
+        - external_request_id
+        - registration_status
+        - recommendation_status
+        - companion
+        - manifest_file_path
+        - usage_outliers
+      properties:
+        _audit:
+          $ref: '#/components/schemas/Audit'
+        uuid:
+          type: string
+          format: uuid          
+        external_request_id:
+          type: string
+          example: 229e7da76708fe374d8c10fa752e72989f      
+        registration_status:
+          $ref: '#/components/schemas/RegistrationStatus'
+        recommendation_status:
+          type: string
+          enum:
+            - success
+            - pgm_error
+          example: success
+        companion:
+          type: array
+          items:
+            $ref: '#/components/schemas/RecommendedPackageData'
+        manifest_file_path:
+          type: string
+          example: pom.xml
+        usage_outliers:
+          type: array
+          items:
+            type: object
+            properties: {}
+      description: Application stack recommendations
+
+    Audit:
+      title: Audit
+      type: object
+      required:
+        - started_at
+        - ended_at
+        - api_version
+      properties:
+        started_at:
+          type: integer
+        ended_at:
+          type: integer
+        api_version:
+          type: string
+
+    StackAggregatorResult:
+      title: StackAggregatorResult
+      type: object
+      required:
+       - _audit
+        - uuid
+        - external_request_id
+        - registration_status
+        - manifest_file_path
+        - manifest_name
+        - ecosystem
+        - unknown_dependencies_count
+        - unknown_dependencies
+        - analyzed_direct_dependencies_count
+        - transitive_count
+        - vulnerable_transitives_count
+        - license_analysis
+        - distinct_licenses
+        - stack_license_conflict
+        - dependencies
```

Comment:
Do we need this if all dependencies are included within the package itself?

Reply:
package doesn't include all dependencies, it includes details(github,vulns) of all direct dependencies and details of vulnerable transitives.

Reply:
Please remove this field